docs/sql-programming-guide.md

```diff
@@ -422,8 +422,8 @@ In the simplest form, the default data source (`parquet` unless otherwise config
 You can also manually specify the data source that will be used along with any extra options
 that you would like to pass to the data source. Data sources are specified by their fully qualified
 name (i.e., `org.apache.spark.sql.parquet`), but for built-in sources you can also use their short
```

Comment:
Thanks for the comment. I am wondering if we could make another jira/PR for that? I prefer not to mix the documentation with code.